robot/EDA/resources/EDA.py

```diff
@@ -331,3 +332,12 @@ def click_edit_on_eda_settings_page(self):
         locator = eda_lex_locators["eda_settings"]["edit"]
         self.selenium.wait_until_page_contains_element(locator, error="Edit button is not available on the page")
         self.selenium.click_element(locator)
+
+    def click_action_button_on_eda_settings_page(self, action):
+        """ Clicks on the action (eg: Save, Cancel) button on the EDA Settings page """
+        locator = eda_lex_locators["eda_settings"]["action"].format(lower(action))
```

Comment:
are u converting action to lowercase ?

Reply:
Yes I had to. Coz the label on the UI says "Save" or "Cancel", whereas the locator is "save" or "cancel".
While we code, we generally use what is visible on the UI, so that's why I am converting to lower